torch/_subclasses/fake_tensor.py

```diff
@@ -0,0 +1,140 @@
+import torch
+
+from torch._subclasses import BaseTensor
+from torch.utils._pytree import tree_map
+from functools import partial
+from torch.fx.operator_schemas import normalize_function
+from typing import Union
+
+aten = torch.ops.aten
+
+_device_not_kwarg_ops = (
+    aten._resize_output_.default,
+    aten.nested_tensor.default,
+    aten.pin_memory.default,
+    aten.is_pinned.default,
+    aten.to.device,
+    aten.to.prim_Device,
+    aten._pin_memory.default,
```

Comment:
@anjali411 Would this be a good operator tag? (Or maybe we should be able to figure this out from schema?)

Reply:
We can already infer this information from the FunctionSchema by iterating through the inputs
```
>>> torch.ops.aten.to.device._schema.arguments
[const Tensor& self, Device device, int dtype, bool non_blocking, bool copy, int? memory_format]
>>> torch.ops.aten.to.device._schema.arguments[1]
Device device
>>> torch.ops.aten.to.device._schema.arguments[1].kwarg_only
False
```

Reply:
Yeaaa I have a [test for that](https://github.com/pytorch/pytorch/blob/master/test/test_fake_tensor.py#L235). Maybe I just should just get rid of the list and move over the checking logic